plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/TestMariaDbLatestConnectorTest.java

```diff
@@ -36,4 +70,383 @@ protected SqlExecutor onRemoteDatabase()
     {
         return server::execute;
     }
+
+    @BeforeClass
+    public void setUpTables()
+    {
+        server.execute("ANALYZE TABLE tpch.nation PERSISTENT FOR ALL;");
+        server.execute("ANALYZE TABLE tpch.region PERSISTENT FOR ALL;");
+        server.execute("ANALYZE TABLE tpch.orders PERSISTENT FOR ALL;");
+        server.execute("ANALYZE TABLE tpch.customer PERSISTENT FOR ALL;");
+    }
+
+    @Override
+    public void testJoinPushdown()
```

Comment:
Why do we need to override these tests?

Reply:
Because, currently MariaDB only collect the number columns(such as int, float, double, datetime, etc) NOT collect the varchar column, the parent test case have some SQL using the varchar column, so rewrite some SQL, so overwrite the test case.